docs/dev/design/new-search-api.rst

```diff
@@ -0,0 +1,238 @@
+           }
+         ]
+       }
+     ],
+     "query": "The final query used in the search",
+     "results": [
+       {
+         "type": "page",
+         "project": {
+           "slug": "docs",
+           "alias": null
+         },
+         "version": {
+           "slug": "latest"
+         },
+         "title": "Main Features",
+         "path": "/en/latest/features.html",
+         "domain": "https://docs.readthedocs.io",
+         "highlights": {
+           "title": []
+         },
+         "blocks": [
+           {
+             "type": "section",
+             "id": "full-text-search",
+             "title": "Full-Text Search",
+             "content": "We provide search across all the projects that we host. This actually comes in two different search experiences: dashboard search on the Read the Docs dashboard and in-doc search on documentation sites, using your own theme and our search results. We offer a number of search features: Search across subprojects Search results land on the exact content you were looking for Search across projects you have access to (available on Read the Docs for Business) A full range of search operators including exact matching and excluding phrases. Learn more about Server Side Search.",
+             "highlights": {
+               "title": [
+                 "Full-<span>Text</span> Search"
+               ],
+               "content": []
+             }
+           },
+           {
+             "type": "domain",
+             "role": "http:post",
+             "name": "/api/v3/projects/",
+             "id": "post--api-v3-projects-",
+             "content": "Import a project under authenticated user. Example request: BashPython$ curl \\ -X POST \\ -H \"Authorization: Token <token>\" https://readthedocs.org/api/v3/projects/ \\ -H \"Content-Type: application/json\" \\ -d @body.json import requests import json URL = 'https://readthedocs.org/api/v3/projects/' TOKEN = '<token>' HEADERS = {'Authorization': f'token {TOKEN}'} data = json.load(open('body.json', 'rb')) response = requests.post( URL, json=data, headers=HEADERS, ) print(response.json()) The content of body.json is like, { \"name\": \"Test Project\", \"repository\": { \"url\": \"https://github.com/readthedocs/template\", \"type\": \"git\" }, \"homepage\": \"http://template.readthedocs.io/\", \"programming_language\": \"py\", \"language\": \"es\" } Example response: See Project details Note Read the Docs for Business, also accepts",
+             "highlights": {
+               "name": [],
+               "content": [
+                 ", json=data, headers=HEADERS, ) print(response.json()) The content of body.json is like,  &quot;name&quot;: &quot;<span>Test</span>"
+               ]
+             }
+           }
+         ]
+       }
+     ]
+   }
+
+Examples
+--------
+
+- ``project:docs project:dev/latest test``: search for ``test`` in the default
+  version of the ``docs`` project, and in the latest version of the ``dev`` project.
```

Comment:
This answered my question above ^^ 👍

Reply:
this one https://github.com/readthedocs/readthedocs.org/pull/9533#discussion_r954316240?

Reply:
Multiple project filters: https://github.com/readthedocs/readthedocs.org/pull/9533#discussion_r954317338